source/renderer.js

```
@@ -27,6 +27,7 @@ function onAction(callback) {
 function sendAction(string) {
     ensureType(string, 'string');
     _registeredActionListeners.forEach(cb => cb(string));
+    return true;
```

To manage the code better, a return state has been inserted here. This helps to find out whether the desired action has actually been sent and has not triggered a silent error.
For example, a query could look like this:
```
const success = sendAction('some.test')
if (success) {
     ...
} else {
     ...
}
```